e2e_playwright/st_toggle_test.py

```diff
@@ -50,14 +50,17 @@ def test_toggle_initial_values(app: Page):
 
 def test_toggle_values_on_click(app: Page):
     """Test that st.toggle updates values correctly when user clicks."""
-    toggle_elements = app.locator(".stCheckbox")
+    toggle_elements = app.get_by_test_id("stCheckbox")
     expect(toggle_elements).to_have_count(8)
 
     for toggle_element in toggle_elements.all():
+        # Not sure if this is needed, but somehow it is slightly
```

Comment:
I think same comment as above: Is there some github link to a video recording as to what is flakey?

Reply:
Also added a bit more info in the comment. Unfortunately, I couldn't find any video to link to while looking through some older runs. And it's hard to trigger this flaky situation :( So, for now, it's probably the best we can do to add the somewhat unspecific guess.